_includes/data/partially_adjudicated.html

```diff
@@ -13,7 +13,7 @@ <h2>
   in our sandbox environment.
 </p>
 
-<a href=/try_the_api.html target="_self" class="in-text__link">Try the BCDA enhancement</a>
```

Comment:
is this the right link? or `/guide.html#try-the-api`? (i believe `guide/try_the_api.html` is the filepath but it's used as part of the `/guide` page)

works: https://stage.bcda.cms.gov/guide.html#try-the-api
doesn't work: https://stage.bcda.cms.gov/guide/try_the_api.html

Reply:
@kyeah Yep, you're right! Sorry, I grabbed the filepath from `{% include guide/try_the_api.html %}` but forgot to format it correctly.